tools/clang/test/CodeGenSPIRV_Lit/spv.intrinsicExecutionModeId.hlsl

```diff
@@ -1,9 +1,9 @@
-// RUN: %dxc -T ps_6_0 -E main -spirv
+// RUN: %dxc -T ps_6_0 -E main -spirv -fcgl -fspv-target-env=vulkan1.1 -Vd %s | FileCheck %s
```

Comment:
Good catch.
Yes, it is required.
This test passes in FileTest.IntrinsicSpirv because spv.intrinsicInstruction.hlsl will set vulkan1.1 at beginning of FileTest.IntrinsicSpirv.
All the runFileTest calls after spv.intrinsicInstruction.hlsl will get vulkan1.1 enabled even though they don't set spv-target-env.

spv.intrinsicExecutionModeId.hlsl is the only shader depends on this behavior.